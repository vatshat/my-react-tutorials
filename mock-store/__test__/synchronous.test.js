import {
    addTodo
} from '../src/synchronous';

// set up redux-mock-store
import configureStore from 'redux-mock-store';
const middlewares = [];
const mockStore = configureStore(middlewares);

it('should add todo', () => {
    const store = mockStore({});

    store.dispatch(addTodo('Feed dog'));

    const actions = store.getActions();
    expect(actions).toMatchSnapshot();
});