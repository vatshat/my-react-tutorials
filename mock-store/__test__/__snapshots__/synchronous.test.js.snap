// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should add todo 1`] = `
Array [
  Object {
    "todo": "Feed dog",
    "type": "ADD_TODO",
  },
]
`;
